data/ar.csv

```diff
@@ -15,11 +15,16 @@
 "Antrag stellen","اقدام الطلب"
 "Antrag auf Eheschließung","طلب عقد زواج"
 "Anwaltskosten","تكاليف المحامي"
+"Anwaltsliste",""
```

Comment:
It seems like `tx pull` does not have an option to skip untranslated entries, but that shouldn't be an issue in our case.